### PR TITLE
[SPARK-49190][K8S][FOLLOW-UP] Add how to populate executor log url vars with env vars

### PR DESCRIPTION
**Pull request by EnricoMi**
### What changes were proposed in this pull request?
Mentions in the docs how to populate custom executor log url vars with environment variable's values.

### Why are the changes needed?
This feature exists since 3.0.0 but is not documented elsewhere.

### Does this PR introduce _any_ user-facing change?
Improves documentation.

### How was this patch tested?
Not tested.

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

File: docs/running-on-kubernetes.md [hunk 1]
```diff
@@ -436,6 +436,13 @@ When there exists a log collection system, you can expose it at Spark Driver `Ex
 spark.ui.custom.executor.log.url='https://log-server/log?appId={{APP_ID}}&execId={{EXECUTOR_ID}}'
 ```
 
+You can add custom variables to this url template, populated with the values of existing executor environment variables like
```
Comment (dongjoon-hyun) [inline docs/running-on-kubernetes.md:439]: `add custom` -> `add additional custom`

File: docs/running-on-kubernetes.md [hunk 2]
```diff
@@ -436,6 +436,13 @@ When there exists a log collection system, you can expose it at Spark Driver `Ex
...
+
+```
+spark.executorEnv.SPARK_EXECUTOR_ATTRIBUTE_YOUR_VAR='$(EXECUTOR_ENV_VAR)'
```
Comment (dongjoon-hyun) [inline docs/running-on-kubernetes.md:442]: `EXECUTOR_ENV_VAR` part looks confusing. According to the above description, `EXISTING_EXECUTOR_ENV_VAR` could be better.

File: docs/running-on-kubernetes.md (hunk 1 above)
Comment (EnricoMi) [reply docs/running-on-kubernetes.md]: Done in e0eba77420a.

File: docs/running-on-kubernetes.md (hunk 2 above)
Comment (EnricoMi) [reply docs/running-on-kubernetes.md]: Sounds good, done in e0eba77420a.

**Review by dongjoon-hyun — APPROVED** [review]
Thank you, @EnricoMi .
Merged to master~

**Comment by EnricoMi** [comment]
Thanks @dongjoon-hyun!

**Pull request closed without merging**